lib/generators/shopify_app/install/install_generator.rb

```diff
@@ -67,7 +67,7 @@ def mount_engine
       def insert_hosts_into_development_config
         inject_into_file(
           'config/environments/development.rb',
-          "  config.hosts = (config.hosts rescue []) << /\\w+\\.ngrok\\.io/\n",
+          "  config.hosts = (config.hosts rescue []) << /.+\\.ngrok\\.io/\n",
```

Comment:
Do we really want to do proper subdomain validation? I think not. We just care it ends with ngrok.io  🔚

Reply:
Also why `(config.hosts rescue []`)?

Reply:
Why not just use `config.hosts.clear` in development env?